types/pdfmake/index.d.ts

```diff
@@ -0,0 +1,70 @@
+// Type definitions for pdfmake 0.1
+// Project: http://pdfmake.org
+// Definitions by: Milen Stefanov <https://github.com/m1llen1um>
+//                 Rajab Shakirov <https://github.com/radziksh>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'pdfmake/build/vfs_fonts' {
+    let pdfMake: {
+        vfs: any;
+        [name: string]: any;
+    };
+}
+
+declare module 'pdfmake/build/pdfmake' {
+    let vfs: TFontFamily;
+    let fonts: { [name: string]: TFontFamilyTypes };
+    function createPdf(documentDefinitions: TDocumentDefinitions): TCreatedPdf;
```

Comment:
Is there a reason why "definitions" is in the plural here?
Even though the test code in this PR (as well as official pdfMake examples) use the singular form (`docDefinition`)